create_infotexts.py

```diff
@@ -233,39 +232,44 @@ def create_commons_filename(self, item):
         :param item: dictionary conatining metadata for one image
         :return: None (populates self.filename)
         """
-        fname_str = ""
-        fname_str += re.sub(" Svenska Cypernexpeditionen\.?", "", item["Beskrivning"])
-        if not fname_str.endswith(" "):
-            fname_str += " "
+        fname_desc = ""
+        fname_desc += re.sub(" Svenska Cypernexpeditionen\.?", "", item["Beskrivning"])
+        if not fname_desc.endswith("."):
+            fname_desc += "."
 
         # Enrich with keywords
+        keywords_to_append = []
         if self.data["keyword_list"]:
             for keyword in self.data["keyword_list"]:
                 if keyword not in item["Beskrivning"]:
-                    fname_str += keyword + " "
+                    keywords_to_append.append(keyword)
+
+        for kw in keywords_to_append:
```

Comment:
it would be simpler to do this as
```python
if keywords_to_append:
    fname_desc += " {}.".format(", ".join(keywords_to_append))
```

Note the space in the start to separate it from the previous info.
I also added a full stop to separate it from the later facts.